func_tests/test_client.py

```diff
@@ -73,3 +75,70 @@ def test_batch_change(vinyldns_test_context):
     assert change2.record.ptrdname == 'change-test.vinyldns.'
     assert change2.type == RecordType.PTR
 
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[0].zone_id, r.changes[0].record_set_id)
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[1].zone_id, r.changes[1].record_set_id)
+
+
+def test_batch_change_review_process_reject(vinyldns_test_context):
+    changes = [
+        AddRecord('test-approve-success.not.loaded.', RecordType.A, 200, AData("4.3.2.1"))
+    ]
+
+    bc = vinyldns_test_context.client.create_batch_change(
+        BatchChangeRequest(changes, 'comments', vinyldns_test_context.group.id)
+    )
+
+    assert bc.status == 'PendingReview'
+    assert bc.approval_status == 'PendingReview'
+    assert len(bc.changes[0].validation_errors) == 1
+
+    vinyldns_test_context.support_client.reject_batch_change(bc.id, 'cannot create the zone')
+
+    completed_bc = vinyldns_test_context.client.get_batch_change(bc.id)
+
+    assert completed_bc.status == 'Rejected'
+    assert completed_bc.approval_status == 'ManuallyRejected'
+    assert completed_bc.reviewer_id == 'support-user-id'
+    assert completed_bc.reviewer_username == 'support-user'
+    assert completed_bc.review_comment == 'cannot create the zone'
+
+
+def test_batch_change_review_process_approve(vinyldns_test_context):
+    approver = vinyldns_test_context.support_client
+
+    changes = [
+        AddRecord('test-approve-success.not.loaded.', RecordType.A, 200, AData("4.3.2.1"))
+    ]
+
+    bc = vinyldns_test_context.client.create_batch_change(
+        BatchChangeRequest(changes, 'comments', vinyldns_test_context.group.id)
+    )
+
+    assert bc.status == 'PendingReview'
+    assert bc.approval_status == 'PendingReview'
+    assert len(bc.changes[0].validation_errors) == 1
+
+    # need to create the zone so the change can succeed
+    zone = {
+        'name': 'not.loaded.',
+        'email': 'test@test.com',
+        'adminGroupId': vinyldns_test_context.group.id,
+        'backendId': 'func-test-backend',
+        'shared': True
+    }
+    zone_create = approver.connect_zone(zone)
```

Comment:
won't be able to do this as .not.loaded is not in `docker/bind9`, needs to be there for this to work

Reply:
copy the zone file for not loaded into `docker/bind9/zones`, then update the file `docker/bind9/etc/named.conf` and put the zone in the conf file